src/Debug/WrappedDelayedListener.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Biig\Component\Domain\Debug;
+
+use Symfony\Component\VarDumper\Caster\ClassStub;
+
+class WrappedDelayedListener
+{
+    /**
+     * @var callable
+     */
+    private $listener;
+
+    /**
+     * @var string
+     */
+    private $pretty;
+
+    private $stub;
```

Comment:
Missing description (which is needed here!)

Reply:
Desciption added